apps/site/assets/ts/stop/components/DisplayTime.tsx

```diff
@@ -121,7 +121,10 @@ const DisplayTime = ({
             )}{" "}
             {/* Prioritize displaying Tomorrow over track name if both are present */}
             <span className="time-details">
-              {tomorrow ? "Tomorrow" : trackName || null}
+              {/* Only show tomorrow if time is not displayed in relative time */}
+              {tomorrow && !(!(isCancelled || isSkipped) && isCR)
```

Comment:
nitpick: is there a clearer way to write this? maybe `tomorrow && !isCR && !(isCanCelled || isSkipped)`

Reply:
Yeah I can change it.  The idea behind this way was it is the `not` of the condition that causes the time to display as relative.